dist/dndbeyond_character.js

```diff
@@ -2346,7 +2364,15 @@ class Beyond20RollRenderer {
                     critical_damage_rolls.push(roll);
                     const dmg_type = critical_damage_types[i];
                     let damage_flags = DAMAGE_FLAGS.REGULAR;
+<<<<<<< HEAD
+<<<<<<< HEAD
                     if (["Healing", "Disciple of Life", "Temp HP", "Alchemical Savant Healing"].includes(dmg_type)) {
+=======
+=======
```

Comment:
I particularly like this change... the conflict is conflicting with the conflict...
![image](https://user-images.githubusercontent.com/27990/94984808-3d44aa00-051e-11eb-8a36-85d52519fcf8.png)